# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../llvm/build-any.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- llvm-10.0.1.src/cmake/config-ix.cmake.vanilla	2020-08-25 12:10:56.564873226 +0200
+++ llvm-10.0.1.src/cmake/config-ix.cmake	2020-08-25 12:11:09.220873240 +0200
@@ -434,7 +434,7 @@
 elseif (LLVM_NATIVE_ARCH MATCHES "riscv64")
   set(LLVM_NATIVE_ARCH RISCV)
 else ()
-  message(FATAL_ERROR "Unknown architecture ${LLVM_NATIVE_ARCH}")
+  set(LLVM_NATIVE_ARCH X86)
 endif ()
 
 # If build targets includes "host", then replace with native architecture.
